ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -255,3 +255,17 @@ SENSOR_ZTM_ZHM_CO2M_VOCM_PMM:
   - CO2M
   - VOCM
   - PMM
+  opt_uses:
+  - soundintensitylevel_sensor
+  - illuminance_sensor
+  - zone_air_aqi_sensor
+  - zone_air_pm10pt0_density_sensor
+  - zone_air_pm2pt5_density_sensor
+
+SENSOR_ZHM_ZOC:
```

Comment:
Is this a stand-alone sensor or does it integrate with a VAV or AHU (or some other piece of equipment)?

Reply:
its a standalone sensor

Reply:
@cstirdivant just got some more information on this sensor, it also measures temperature.

Also, the zone_occupant_counter is a total for the zone visible to the sensor, however that entire zone can be split into different sections which can have their own individual counter. We're only using the total count, so should this just be left as is, or what would be best way to proceed?

Reply:
I suggest using only the total count of occupants